src/Server.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file
+ * @brief Config file
+ * @author LionZXY
+ * @project Recast
+ * @date 08.06.17
+ *
+ * Server file description
+ *
+ **/
+
+#include <boost/log/trivial.hpp>
+#include <boost/property_tree/json_parser.hpp>
+
+#include <boost/log/expressions.hpp>
+#include <boost/log/sinks/text_file_backend.hpp>
+#include <boost/log/utility/setup/file.hpp>
+#include <boost/log/utility/setup/common_attributes.hpp>
+#include <boost/log/utility/setup/console.hpp>
+#include <boost/filesystem.hpp>
+
+#include "./headers/Server.h"
+
+void initLogger() {
+    std::string output = (std::string("recast_") + std::to_string(time(NULL)) + std::string(".log"));
+    boost::filesystem::path dir("./logs/");
+    if (!boost::filesystem::exists(dir))
+        if (boost::filesystem::create_directory(dir)) {
+            BOOST_LOG_TRIVIAL(info) << "Папка " << dir << " создана успешно";
```

Comment:
Cyrillic symbols in source code is bad practice. If you want localize this, you should use some resource file. OR just header with constants.